tests/scripts/custodian/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM ubuntu:20.04
+WORKDIR /usr/app/src
+
+ARG AWS_ACCESS_KEY_ID
+ENV AWS_ACCESS_KEY_ID=$AWS_ACCESS_KEY_ID
+ARG AWS_SECRET_ACCESS_KEY
+ENV AWS_SECRET_ACCESS_KEY=$AWS_SECRET_ACCESS_KEY
+ARG AZURE_SUBSCRIPTION_ID
+ENV AZURE_SUBSCRIPTION_ID=$AZURE_SUBSCRIPTION_ID
+ARG AZURE_TENANT_ID
+ENV AZURE_TENANT_ID=$AZURE_TENANT_ID
+ARG AZURE_CLIENT_ID
+ENV AZURE_CLIENT_ID=$AZURE_CLIENT_ID
+ARG AZURE_CLIENT_SECRET
+ENV AZURE_CLIENT_SECRET=$AZURE_CLIENT_SECRET
+ARG RANCHER_GKE_CREDENTIAL
+ENV RANCHER_GKE_CREDENTIAL=$RANCHER_GKE_CREDENTIAL
+ARG RANCHER_LINODE_CREDENTIAL
+ENV RANCHER_LINODE_CREDENTIAL=$RANCHER_LINODE_CREDENTIAL
+ARG TAG_USER_KEYS
+ENV TAG_USER_KEYS=$TAG_USER_KEYS
+
+ENV CUSTODIAN_YAML=instances.yaml
+
+RUN apt-get update && apt-get install -y python3-venv python3-pip
+RUN python3 -m venv custodian-venv
+RUN . custodian-venv/bin/activate
+RUN pip3 install c7n c7n_azure c7n_gcp c7n_mailer
+
+
+ADD ./* ./
+
+# get user keys, replace holder params in yaml files appropriately
+SHELL ["/bin/bash", "-c"] 
+
+RUN if [[ -z "$TAG_USER_KEYS" ]] ; then while read -r line; do export USER_KEYS="${USER_KEYS}|$line"; done < user-keys.txt; \
+    else export USER_KEYS=$TAG_USER_KEYS; fi
+RUN for i in `ls *.yaml`; do cat $i | sed -e 's/USERKEYS/'"$USER_KEYS"'/g' -i $i; done
+RUN while read -r line; do export DO_NOT_DELETE_KEYS="${DO_NOT_DELETE_KEYS}|$line"; done < do-not-delete-keys.txt
+RUN for i in `ls *.yaml`; do cat $i | sed -e 's/DONOTDELETEKEYS/'"$DO_NOT_DELETE_KEYS"'/g' -i $i; done
```

Comment:
Same comment as above.

Reply:
yes, this is necessary. thanks for checking